docs/kb/release-notes.md

```diff
@@ -0,0 +1,29 @@
+# Release Notes
+
+Here you'll find the latest updates on what's changed in the Pardot API and API documentation.
+
+## March 2017
+
+### Visitor Activities 
+
+A few updates were made to the `query` endpoint for both version 3 and 4. These include:
+
+* any email activity that is part of an email list will now include the list_email_id
+* any email activity that belongs to an email which used an email template, will include the email_template_id
```

Comment:
'related to' instead of 'belongs to' and 'generated from' instead of 'which used'